config/dev.exs

```diff
@@ -3,10 +3,10 @@ import Config
 # Configure your database
 if System.get_env("DB_TYPE") == "postgresql" do
   config :starknet_explorer, StarknetExplorer.Repo,
-    database: "madaraexplorer_dev",
-    username: "madaraexplorer_user",
-    password: "madaraexplorerlambda",
-    hostname: "localhost",
+    database: System.get_env("DB_NAME"),
+    username: System.get.env("DB_USER"),
+    password: System.get.env("DB_PASS"),
+    hostname: System.get.env("DB_HOST"),
```

Comment:
This should be:
```suggestion
    database: System.get_env("DB_NAME"),
    username: System.get_env("DB_USER"),
    password: System.get_env("DB_PASS"),
    hostname: System.get_env("DB_HOST"),
```

And please add the old values as the default in case that those env vars aren't set. You can do that by passing the values as a second parameter to `System.get_env`.

Reply:
Fixed